utilities/cli/tests/test_holoscan_cli_consolidation.py

```
@@ -0,0 +1,121 @@
+    # holoscan-cli#177 decoupled the container base image from the CLI version:
+    # `run`/`build`/`package` fatal unless a base SDK version (or --base-img) is
+    # configured. The ./holohub wrapper sets this; the direct `python -m
+    # holoscan_cli` invocations below must supply it too. Mirror the HoloHub
+    # default SDK version so the dryrun resolves a base image.
+    env.setdefault("HOLOSCAN_CLI_BASE_SDK_VERSION", "4.3.0")
+    return env
+
+
+def _run_holoscan_cli(*args: str) -> subprocess.CompletedProcess:
+    env = _holoscan_cli_env()
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)